src/Method.php

```diff
@@ -251,35 +251,37 @@ protected function normalizeParams(DocBlock $phpdoc)
      */
     protected function normalizeReturn(DocBlock $phpdoc)
     {
-        //Get the return type and adjust them for beter autocomplete
+        //Get the return type and adjust them for better autocomplete
         $returnTags = $phpdoc->getTagsByName('return');
-        if ($returnTags) {
-            /** @var ReturnTag $tag */
-            $tag = reset($returnTags);
-            // Get the expanded type
-            $returnValue = $tag->getType();
-
-            // Replace the interfaces
-            foreach ($this->interfaces as $interface => $real) {
-                $returnValue = str_replace($interface, $real, $returnValue);
-            }
-
-            // Set the changed content
-            $tag->setContent($returnValue . ' ' . $tag->getDescription());
-            $this->return = $returnValue;
 
-            if ($tag->getType() === '$this') {
-                Str::contains($this->root, Builder::class)
-                    ? $tag->setType($this->root . '|static')
-                    : $tag->setType($this->root);
-            }
-        } else {
+        if (count($returnTags) === 0) {
```

Comment:
IMHO this is fine, it's PHP after all and it's not ambiguous in this context:
```suggestion
        if (!$returnTags) {
```